CHANGELOG.md

```diff
@@ -304,6 +550,30 @@ Mark  items as `Added`, `Changed`, `Fixed`, `Modified`, `Removed`, `Untested Fea
         Refer the [forms-flow-ai-micro-front-ends](https://github.com/AOT-Technologies/forms-flow-ai-micro-front-end) repository for further details.
 * Dashboard authorization is moved from designer role to admin user.
 
+#### <ins>Premium Features </ins>
+
+`Added`
+
+**forms-flow-web**
+
+* Added `form bundling` feature as a premium feature, refer [here](https://aot-technologies.github.io/forms-flow-ai-doc/#formBundling) for more details.
+
+**forms-flow-bpm**
+
+* Added CombineSubmissionBundleListener to support form bundling feature.
+* Added RequestStateListener to support Request status.
```

Comment:
Links to the listeners to be added when merged to EE. Since CombineSubmissionBundleListener, RequestStateListener  readme are currently available in EE